modules/storage/ceph/types.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 Red Hat
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ceph
+
+// Defaults is used as type to reference defaults
+type Defaults string
+
+// Spec is the Cinder Ceph Backend struct defining defaults
+type Spec struct {
+	Defaults *Defaults
+}
```

Comment:
I can't find where this is used.  Is it something that would be imported elsewhere externally?

Reply:
oh thanks, good catch, since now the module is shared and I reworked it, I don't need that part anymore.